### PR TITLE
address spencer's comments

### PR DESCRIPTION
**Pull request by goelvidhi**

### CODE REVIEW

**Review by lisongxu — DISMISSED** [review]

File: draft-ietf-tcpm-rfc8312bis.md [hunk 1]
```diff
@@ -197,6 +197,15 @@ algorithms, this document updates the first paragraph of {{Section 3 of
 in {{Section 3 of !RFC5033}}, which allows for CUBIC's behavior as defined
 in this document.
 
+Specifically, CUBIC may increase congestion window more aggressively than
```
Comment (larseggert) [inline draft-ietf-tcpm-rfc8312bis.md:200]:
```suggestion
Specifically, CUBIC may increase the congestion window more aggressively than
```

File: draft-ietf-tcpm-rfc8312bis.md [hunk 2]
```diff
@@ -197,6 +197,15 @@ algorithms, this document updates the first paragraph of {{Section 3 of
...
+Reno during the congestion avoidance phase. According to {{!RFC5681}},
+during congestion avoidance, sender must not increment cwnd by more than
```
Comment (larseggert) [inline draft-ietf-tcpm-rfc8312bis.md:202]:
```suggestion
during congestion avoidance, the sender must not increment cwnd by more than
```

File: draft-ietf-tcpm-rfc8312bis.md [hunk 3]
```diff
@@ -197,6 +197,15 @@ algorithms, this document updates the first paragraph of {{Section 3 of
...
+SMSS bytes once per RTT whereas CUBIC may increase cwnd much more
```
Comment (larseggert) [inline draft-ietf-tcpm-rfc8312bis.md:203]:
```suggestion
SMSS bytes once per RTT, whereas CUBIC may increase cwnd much more
```

File: draft-ietf-tcpm-rfc8312bis.md [hunk 4]
```diff
@@ -197,6 +197,15 @@ algorithms, this document updates the first paragraph of {{Section 3 of
...
+aggressively than SMSS bytes per RTT. Additionally, CUBIC recommends HyStart++
```
Comment (larseggert) [inline draft-ietf-tcpm-rfc8312bis.md:204]:
```suggestion
aggressively. Additionally, CUBIC recommends the HyStart++
```

File: draft-ietf-tcpm-rfc8312bis.md [hunk 5]
```diff
@@ -197,6 +197,15 @@ algorithms, this document updates the first paragraph of {{Section 3 of
...
+algorithm for slow start which allows for cwnd increases of more than SMSS
```
Comment (larseggert) [inline draft-ietf-tcpm-rfc8312bis.md:205]:
```suggestion
algorithm {{!I-D.ietf-tcpm-hystartplusplus}} for slow start, which allows for cwnd increases of more than SMSS
```

File: draft-ietf-tcpm-rfc8312bis.md [hunk 6]
```diff
@@ -197,6 +197,15 @@ algorithms, this document updates the first paragraph of {{Section 3 of
...
+bytes for incoming acknowledgments during slow start while this behavior
```
Comment (larseggert) [inline draft-ietf-tcpm-rfc8312bis.md:206]:
```suggestion
bytes for incoming acknowledgments during slow start, while this behavior
```

File: draft-ietf-tcpm-rfc8312bis.md [hunk 7]
```diff
@@ -302,13 +311,13 @@ synchronizations.
 ## Principle 2 for Reno-Friendliness
 
 CUBIC promotes per-flow fairness to Reno. Note that Reno
-performs well over paths with short RTTs and small bandwidths (or
-small BDPs). There is only a scalability problem in networks with long
-RTTs and large bandwidths (or large BDPs).
+performs well over paths with small bandwidth-delay products, and only
+experiences problems when attempting to increase bandwidth utilization
+on paths with large bandwidth-delay products.
```
Comment (larseggert) [inline draft-ietf-tcpm-rfc8312bis.md:316]: Didn't we have a long discussion earlier that resulted in this text? Would not like to reopen that can of worms.

Comment (goelvidhi) [reply draft-ietf-tcpm-rfc8312bis.md:316]: Can you remind me about the previous discussion (my memory is failing me)? AFAICS, this text is same as the original RFC 8312.

Comment (larseggert) [reply draft-ietf-tcpm-rfc8312bis.md:317]: Maybe I am misremembering.

File: draft-ietf-tcpm-rfc8312bis.md [hunk 8]
```diff
@@ -302,13 +312,13 @@ synchronizations.
...
```
Comment (larseggert) [inline draft-ietf-tcpm-rfc8312bis.md:317]:
```suggestion
experiences problems when transmitting over
paths with large bandwidth-delay products.
```

**Review by larseggert — APPROVED** [review]

**Review by lisongxu — DISMISSED** [review]

File: draft-ietf-tcpm-rfc8312bis.md (hunk 8 above)
Comment (goelvidhi) [reply draft-ietf-tcpm-rfc8312bis.md:317]: This text is about poor scalability of Reno on large BDP paths, so we ought to mention bandwidth utilization in some form. Suggestion, `when attempting to fully utilize the bandwidth on paths with large`?

**Review by larseggert — APPROVED** [review]

**Pull request merged**